src/output.cpp

```diff
@@ -1041,6 +1041,43 @@ void TextOutput::helper_error(int retcode, const HelperErrorInfo &info) const
       << ", retcode: " << retcode;
 }
 
+void TextOutput::benchmark_results(
+    const std::map<std::string, uint32_t> &results) const
+{
+  if (results.empty()) {
+    return;
+  }
+
+  const std::string BENCHMARK = "BENCHMARK";
+  const std::string NANOSECONDS = "NANOSECONDS";
```

Comment:
should these be file level static consts?

Reply:
So that this isn't blocked on https://github.com/bpftrace/bpftrace/pull/4356, I can hand-code the `ns` suffix for now and switch to `DisplayUnit` when that lands. WDYT?

Reply:
I'll submit and you can use it if you like. I'm not concerned either way.

Reply:
I ended up using `util::duration_str` to generate the time string.